diff --git a/node_modules/next/dist/esm/lib/metadata/is-metadata-route.js b/node_modules/next/dist/esm/lib/metadata/is-metadata-route.js
index ee7dc38..a680434 100644
--- a/node_modules/next/dist/esm/lib/metadata/is-metadata-route.js
+++ b/node_modules/next/dist/esm/lib/metadata/is-metadata-route.js
@@ -61,7 +61,7 @@ export function isMetadataRouteFile(appDirRelativePath, pageExtensions, withExte
     const metadataRouteFilesRegex = [
         new RegExp(`^[\\\\/]robots${withExtension ? `\\.${getExtensionRegexString(pageExtensions.concat("txt"))}$` : ""}`),
         new RegExp(`^[\\\\/]manifest${withExtension ? `\\.${getExtensionRegexString(pageExtensions.concat("webmanifest", "json"))}$` : ""}`),
-        new RegExp(`^[\\\\/]favicon\\.ico$`),
+        new RegExp(`^[\\\\/]_NO_favicon\\.ico$`),
         new RegExp(`[\\\\/]sitemap${withExtension ? `\\.${getExtensionRegexString(pageExtensions.concat("xml"))}$` : ""}`),
         new RegExp(`[\\\\/]${STATIC_METADATA_IMAGES.icon.filename}\\d?${withExtension ? `\\.${getExtensionRegexString(pageExtensions.concat(STATIC_METADATA_IMAGES.icon.extensions))}$` : ""}`),
         new RegExp(`[\\\\/]${STATIC_METADATA_IMAGES.apple.filename}\\d?${withExtension ? `\\.${getExtensionRegexString(pageExtensions.concat(STATIC_METADATA_IMAGES.apple.extensions))}$` : ""}`),
diff --git a/node_modules/next/dist/lib/metadata/is-metadata-route.js b/node_modules/next/dist/lib/metadata/is-metadata-route.js
index e4d9d2f..0a5c1e0 100644
--- a/node_modules/next/dist/lib/metadata/is-metadata-route.js
+++ b/node_modules/next/dist/lib/metadata/is-metadata-route.js
@@ -81,7 +81,7 @@ function isMetadataRouteFile(appDirRelativePath, pageExtensions, withExtension)
     const metadataRouteFilesRegex = [
         new RegExp(`^[\\\\/]robots${withExtension ? `\\.${getExtensionRegexString(pageExtensions.concat("txt"))}$` : ""}`),
         new RegExp(`^[\\\\/]manifest${withExtension ? `\\.${getExtensionRegexString(pageExtensions.concat("webmanifest", "json"))}$` : ""}`),
-        new RegExp(`^[\\\\/]favicon\\.ico$`),
+        new RegExp(`^[\\\\/]_NO_favicon\\.ico$`),
         new RegExp(`[\\\\/]sitemap${withExtension ? `\\.${getExtensionRegexString(pageExtensions.concat("xml"))}$` : ""}`),
         new RegExp(`[\\\\/]${STATIC_METADATA_IMAGES.icon.filename}\\d?${withExtension ? `\\.${getExtensionRegexString(pageExtensions.concat(STATIC_METADATA_IMAGES.icon.extensions))}$` : ""}`),
         new RegExp(`[\\\\/]${STATIC_METADATA_IMAGES.apple.filename}\\d?${withExtension ? `\\.${getExtensionRegexString(pageExtensions.concat(STATIC_METADATA_IMAGES.apple.extensions))}$` : ""}`),
